test/runtests.jl

```diff
@@ -4,7 +4,8 @@ using TestSetExtensions
 tests = [
     "UCI",
     "GitHub",
-    "DataDryad"    
+    "DataDryad",
+    "DataDepsTest"    
```

Comment:
Maybe "Integration Tests"  would be a better name?

Reply:
Sure, I'll change it.